apps/web/components/shared/kyc/kyc-4/file-upload-area.tsx

```
@@ -0,0 +1,91 @@
+			onClick={() => {
+				if (isProcessing || !fileUploadRef.current) return
+				fileUploadRef.current.click()
+			}}
+			onKeyUp={(e) => {
+				if (isProcessing || !fileUploadRef.current) return
+				if (e.key === 'Enter' || e.key === ' ') {
+					fileUploadRef.current.click()
+				}
+			}}
+			tabIndex={0}
+			role="button"
```

Biome (1.9.4)

[error] The elements with the following roles can be changed to the following elements:
<button>
<input type="button">


For examples and more information, see WAI-ARIA Roles

(lint/a11y/useSemanticElements)